MediaAdminBundle/Resources/public/less/mediaModal.less

```diff
@@ -36,6 +36,14 @@
                     ul {
                         padding-left: 10px;
 
+                        a {
```

Comment:
these properties isn't necessary, add only a tag span in menu_macros_modal.html.twig 

```
    <a
        id="media-modal-{{ folder.id }}"
        class="media-modal-menu-folder"
        data-url="{{ path('open_orchestra_api_media_list', {'folderId': folder.id}) }}"
        data-displayed-elements="name"
    >
        <span>{{ folder.name }}</span>
    </a>
```
and you will have same result, because these properties are already set in nav.less